README.md

```diff
@@ -40,6 +40,7 @@ Features
     and Geth's
     [Personal](https://github.com/ethereum/go-ethereum/wiki/Management-APIs#personal)
     client APIs
+-   Support for [Alchemy](https://docs.alchemyapi.io/alchemy/guides/getting-started#web-3-j), so you don't have to run an Ethereum client yourself 
```

Comment:
Please can you  put Alchemy and Infura on the same line so we're not repeating ourselves?